nbconvert/templates/latex/base.tplx

```diff
@@ -13,7 +13,29 @@ This template does not define a docclass, the inheriting class must define this.
     ((* block docclass *))((* endblock docclass *))
     
     ((* block packages *))
-    \usepackage{graphicx} % Used to insert images
+    % Nicer default font than Computer Modern for most use cases
+    \usepackage{palatino}
+
+    % Basic figure setup, for now with no caption control since it's done
+    % automatically by Pandoc (which extracts ![](path) syntax from Markdown).
+    \usepackage{graphicx}
+    % We will generate all images so they have a width \maxwidth. This means
+    % that they will get their normal width if they fit onto the page, but
+    % are scaled down if they would overflow the margins.
```

Comment:
This should already be handled line 204 and works correctly for me with pandoc 1.15.2 (with a default of min witdth 80% pagewidth, height 80% pageheight,to ensure extra-long image still fit.) So This would be a regression.